src/examples/debugtext.d

```diff
@@ -1,7 +1,7 @@
 //------------------------------------------------------------------------------
-//  debugtext_print.d
+//  debugtext.d
 //
-//  Demonstrates formatted printing with sokol.debugtext
+//  Test sokol_debugtext.h
 //------------------------------------------------------------------------------
 module examples.debugtext_print;
```

Comment:
Missing rename `module` name. Affect wasm32 target.

https://github.com/kassane/sokol-d/blob/c20da5b0ad4e3e6d42da84ff92b0349297cead53/build.zig#L557

```bash
zig build run-debugtext -Dtarget=wasm32-emscripten -Doptimize=ReleaseSmall
run-debugtext
└─ run /home/kassane/.cache/zig/p/12203d61e8947f25993a86b4d73e2d91b4c1d96f1aa07c581b7dffba7cb5ca91bc35/upstream/emscripten/emrun
   └─ emcc
      └─ zig build-obj debugtext ReleaseSmall wasm32-emscripten 1 errors
error: unable to check cache: stat file '/home/kassane/sokol-d/zig-cache/o/7ba9215d07edf97684e70f3615ac16da/examples.debugtext.o' failed: FileNotFound
```

Reply:
ah oops, thanks I missed that.

Reply:
I'll do a search and replace, missed a lot of other places (dub.sdl etc)

Reply:
should be fixed now